### ISSUE TITLE
Turn off page-swipe within a defined area of page?

### ISSUE THREAD

**Issue by CatherineEng** [Simbul/baker#459]
Is it possible to reference the "-baker-page-turn-swipe" variable from within one of the book's .html or .css files?

I am trying to delineate an area on one of my book's pages that will not respond to the page-turn-swipe parameter value set in the book.json file (set to "true" there) - so that a swipe gesture within that defined area will not take the viewer to the next page but rather allow a bit of drawing query to work there instead. I'm trying to create a 'doodle area' on the page i.e. a spot where readers can draw or color using the swipe gestures. Currently the drawing query does not work smoothly because the swipes change the page rather than draw on it.

Something like this:

```
    .someStyle {
        position: relative;
        top: 250px;
        left: 85px;
        width: 200px;
        height:100px;
        -baker-page-turn-swipe:false;
    }
```

Thanks!

**Comment by folletto** [comment]
It's not possible, we are sorry. However, we take this in as a feature request: we need to check, but it should be possible to be implemented in the future in JavaScript.

**Comment by CatherineEng** [comment]
Hi Davide,

Thank you very much for your reply. 

I think your idea to implement something using Javascript  will work - after I contacted you, I was able to implement an area without swipe using William Malone's drawing app - it uses javascript to define an area for the paint canvas, and his method effectively creates a no-swipe zone within the coordinates / width / height that are specified... see lines 24 and 25 for the canvas; 51-54 for the drawing area; and from line 644 onwards where the canvas element is drawn on the page. 

And also - thank you so very much for Baker. It is an incredible framework, and all of your ingenuity and creativity and generosity is doing a tremendous amount  towards pushing mobile applications to the next level.

Best,
Catherine

**Comment by folletto** [comment]
Thanks!

That's great news. :)
And feel free to share your findings with the community. :)

**Comment by folletto** [comment]
As a reference, I think it's this app you were talking about:
http://www.williammalone.com/articles/create-html5-canvas-javascript-drawing-app/

I close this issue, since the original problem seems solved and we have a similar request in Issue #466 that will address this as well if needed. :)
